tickets/src/middlewares/web3Middleware.js

```diff
@@ -134,6 +137,44 @@ const web3Middleware = config => {
             action.transaction
           )
         }
+
+        const keyId = `${lockAddress}-${accountAddress}`
+        if (action.type === UPDATE_TRANSACTION) {
+          const existingTransaction = getState().transactions[action.hash]
+          if (
```

Comment:
I wonder if we can take this boolean logic and name it, so that the `if` would read a little more clearly.

Reply:
Fair - although it does have a comment directly beneath to indicate purpose?

Reply:
Reasonable